### PR TITLE
Feature/cleanup amp fixes

### PR DESCRIPTION
**Pull request by hugowetterberg**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -51,14 +51,20 @@ Db.prototype._getConnection = function (cb) {
 // * **cb** function (err, result, fields)
 //
 Db.prototype.select = function select(query, data, cb) {
+  // Allow optional data param
+  if (typeof data === 'function') {
+    data = [];
+    cb = data;
```
Comment (voxpelli) [inline index.js:56]: You probably want to switch these two lines

File: package.json [hunk 2]
```diff
@@ -1,6 +1,7 @@
 {
   "name": "blmysql",
   "version": "1.0.0",
+  "private": true,
```
Comment (voxpelli) [inline package.json:4]: :+1:

File: index.js (hunk 1 above)
Comment (hugowetterberg) [inline index.js:56]: Hah, yeah, that might be a good idea ;)

**Comment by voxpelli** [comment]
LGTM

**Pull request merged**